tsup.config.ts

```diff
@@ -0,0 +1,19 @@
+import type { Options } from 'tsup';
+
+const env = process.env.NODE_ENV;
+
+export const tsup: Options = {
```

Comment:
is it pretty much the same as doing
```
tsup src/index.ts --platform node --format esm,cjs --dts --out-dir dist
```
right?

also, we need a bundle for browser format, like 
```
tsup src/index.ts --platform browser --format iife --global-name aptosSDK --minify --out-dir dist/browser
```